src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Arkivmelding.cs

```diff
@@ -0,0 +1,412 @@
+            Type = type;
+            Text = text;
+        }
+
+        /// <summary>
+        ///  Gets or sets the SystemID
+        /// </summary>
+        [XmlElement(ElementName = "systemID")]
+        public string SystemID { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetDato
+        /// </summary>
+        [XmlElement(ElementName = "opprettetDato")]
+        public DateTime OpprettetDato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetAv
+        /// </summary>
+        [XmlElement(ElementName = "opprettetAv")]
+        public string OpprettetAv { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the ArkivertDato
+        /// </summary>
+        [XmlElement(ElementName = "arkivertDato")]
+        public DateTime ArkivertDato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the ArkivertAv
+        /// </summary>
+        [XmlElement(ElementName = "arkivertAv")]
+        public string ArkivertAv { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the ReferanseForelderMappe
+        /// </summary>
+        [XmlElement(ElementName = "referanseForelderMappe")]
+        public string ReferanseForelderMappe { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Dokumentbeskrivelse
+        /// </summary>
+        [XmlElement(ElementName = "dokumentbeskrivelse")]
+        public Dokumentbeskrivelse Dokumentbeskrivelse { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Tittel
+        /// </summary>
+        [XmlElement(ElementName = "tittel")]
+        public string Tittel { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OffentligTittel
+        /// </summary>
+        [XmlElement(ElementName = "offentligTittel")]
+        public string OffentligTittel { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Journalposttype
+        /// </summary>
+        [XmlElement(ElementName = "journalposttype")]
+        public string Journalposttype { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Journalstatus
+        /// </summary>
+        [XmlElement(ElementName = "journalstatus")]
+        public string Journalstatus { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Journaldato
+        /// </summary>
+        [XmlElement(ElementName = "journaldato")]
+        public DateTime Journaldato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Korrespondansepart
+        /// </summary>
+        [XmlElement(ElementName = "korrespondansepart")]
+        public Korrespondansepart Korrespondansepart { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Type
+        /// </summary>
+        [XmlAttribute(AttributeName = "type")]
+        public string Type { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Text
+        /// </summary>
+        [XmlText]
+        public string Text { get; set; }
```

Comment:
Text instead of teskt ? Should be consistent if the other properties are in Norwegian.